code/game/objects/items/singularityhammer.dm

```diff
@@ -85,6 +85,9 @@
 	throwforce = 30
 	throw_range = 7
 	w_class = WEIGHT_CLASS_HUGE
+	var/zap_power = 20000
```

Comment:
these don't look like they're used anymore

Reply:
you right, they gone now